samples/bookinfo/src/productpage/Dockerfile

```diff
@@ -32,10 +32,7 @@ ENV FLOOD_FACTOR ${flood_factor:-0}
 EXPOSE 9080
 WORKDIR /opt/microservices
```

Comment:
can be deleted?

Reply:
Since the files are still located in `/opt/microservices` we can't/shouldn't remove the workdir.